server/src/repositories/notification.repository.ts

```diff
@@ -2,6 +2,8 @@ import { Inject, Injectable } from '@nestjs/common';
 import { render } from '@react-email/render';
 import { createTransport } from 'nodemailer';
 import React from 'react';
+import AlbumInviteEmail from 'src/emails/album-invite.email';
+import AlbumUpdateEmail from 'src/emails/album-update.email';
```

Comment:
can you make these work like WelcomeEmail where it is a named export instead of a default export?